spec/api/api.yaml

```diff
@@ -1044,7 +1050,7 @@ components:
             example: ['people[22].address', 'people[3]']
         updatedBlobs:
           description: A dictionary whose keys are the key paths into the document to be updated with the
-            blob objects and values are the blob keys in the dataset.
+            blob objects and values are the urls from which to obtain the blob files.
           type: object
           additionalProperties: {}
           example: {'photo.thumbnail': 's1.jpg'}
```

Comment:
is this correct if this is a url? `s1.jpg` doesn't look like a url to me.

Reply:
Yes.  I missed one.  Will fix.

Reply:
fixed